os/services/rpl-border-router/Makefile.rpl-border-router

```diff
@@ -0,0 +1,10 @@
+CFLAGS += -DBUILD_WITH_RPL_BORDER_ROUTER=1
```

Comment:
I don't see the need for this. It strikes me as though this could be controlled by the pre-processor through `-conf.h`

Reply:
Scrap the comment above.

Reply:
No I think you were right weren't you?

Reply:
My use-case here was an example where a user may wish to enable/disable the module depending on circumstances. It would be nice to be able to do this through header files to take advantage of dependency generation. This would mean that the example's makefile would always include the module in the build, only we'd get rid of it at link time.

This of course cannot work unless we know the module can always compile.